osu.Game/Screens/Play/PlayerSettings/VisualSettings.cs

```diff
@@ -23,15 +24,15 @@ public VisualSettings()
             {
                 new OsuSpriteText
                 {
-                    Text = "Background dim:"
+                    Text = GameplaySettingsStrings.BackgroundDim
                 },
                 dimSliderBar = new PlayerSliderBar<double>
                 {
                     DisplayAsPercentage = true
                 },
                 new OsuSpriteText
                 {
-                    Text = "Background blur:"
+                    Text = GameplaySettingsStrings.BackgroundBlur
```

Comment:
Of note, a trailing colon is dropped here, but it looks better without so this is fine.